pa11y_scripts/.crt-view.json

```diff
@@ -1,19 +1,25 @@
 {
-    "urls": [
-        {
-          "useIncognitoBrowserContext": true,
-          "url": "http://127.0.0.1:8000/form/view/",
-          "actions": [
-            "wait for url to be http://127.0.0.1:8000/accounts/login/?next=/form/view/",
-            "set field #id_username to pa11y_tester",
-            "set field #id_password to imposing40Karl5monomial",
-            "click #login",
-            "wait for url to be http://127.0.0.1:8000/form/view/",
-            "wait for element #id_assigned_to to be visible",
-            "click element #submitted-sort",
-            "wait for url to be http://127.0.0.1:8000/form/view/?sort=create_date"
-          ],
-          "screenCapture": "pa11y-screenshots/crt-sort-view.png"
-        }
-    ]
-}
+  "urls": [
+    {
+      "useIncognitoBrowserContext": true,
+      "url": "http://127.0.0.1:8000/form/view/",
+      "actions": [
+        "wait for url to be http://127.0.0.1:8000/accounts/login/?next=/form/view/",
+        "set field #id_username to pa11y_tester",
+        "set field #id_password to imposing40Karl5monomial",
+        "click #login",
+        "wait for url to be http://127.0.0.1:8000/form/view/",
```

Comment:
CircleCI firsts fails while timing out waiting for this statement to resolve. We should correct the URL here or use `path` https://github.com/pa11y/pa11y#wait-for-fragmentpathurl